v2.0/alter-user.md

```diff
@@ -0,0 +1,58 @@
+---
+title: ALTER USER
+summary: The ALTER USER statement can be used to add or change a user's password.
+toc: false
+---
+
+<span class="version-tag">New in v2.0:</span> The `ALTER USER` [statement](sql-statements.html) can be used to add or change a [user's](create-and-manage-users.html) password.
```

Comment:
Let's use this formulation: 

> `ALTER USER` is a subcommand of the [`ALTER TABLE`](alter-table.html) statement that is used to add or change a [user's](create-and-manage-users.html) password.

We'll eventually want to update the pages for all such subcommands to use the same pattern.

Reply:
ALTER USER is not a subcommand -- it is a separate statement.

Reply:
Ugh, of course. Sorry about that. Head stretched in too many directions.